politeiawww/proposals_test.go

```diff
@@ -477,24 +462,14 @@ func TestValidateProposal(t *testing.T) {
 				ErrorCode: www.ErrorStatusProposalDuplicateFilenames,
 			}},
 
-		{"too may md files", *propMaxMDFiles, usr,
-			www.UserError{
-				ErrorCode: www.ErrorStatusMaxMDsExceededPolicy,
-			}},
-
-		{"too many images", *propMaxImages, usr,
-			www.UserError{
-				ErrorCode: www.ErrorStatusMaxImagesExceededPolicy,
-			}},
-
-		{"md file too large", *propMDLarge, usr,
+		{"too many attached files", *propMaxAttachments, usr,
 			www.UserError{
-				ErrorCode: www.ErrorStatusMaxMDSizeExceededPolicy,
+				ErrorCode: www.ErrorStatusMaxAttachmentsExceededPolicy,
 			}},
 
-		{"image too large", *propImageLarge, usr,
+		{"index MD file too large", *propIndexMDLarge, usr,
 			www.UserError{
-				ErrorCode: www.ErrorStatusMaxImageSizeExceededPolicy,
+				ErrorCode: www.ErrorStatusMaxIndexFileSizeExceededPolicy,
 			}},
```

Comment:
You need to add unit tests to fully cover this functionality.

too many index files
index file too large
too many attachment files
attachment file too large
duplicate attachment
attachment is duplicate of index

CMS unit tests need to be added to cover this new functionality as well.